R/repr_recordedplot.r

```diff
@@ -1,4 +1,21 @@
 is_cairo_installed <- function() requireNamespace('Cairo', quietly = TRUE)
+is_ragg_installed <- function() requireNamespace('ragg', quietly = TRUE)
+
+get_device <- function(ragg, Cairo, grDevices) {
+	backends <- getOption('repr.plot.backends')
+	for (o in backends) switch(
+		o,
+		ragg = if (is_ragg_installed()) return(ragg),
+		Cairo = if (is_cairo_installed()) return(Cairo),
+		grDevices = return(grDevices),
+		stop("Unknown entry in getOption('repr.plot.backends'): ", o)
+	)
+	stop(
+		"No plotting devide found in getOption('repr.plot.backends'): ",
```

Comment:
Type: devide should be device.

Since this is an message intended for end users: maybe it could be even more explicit and say something like
```
stop(
"None of the backends specified in getOption('repr.plot.backends'): ",
toString(backends):
"seems to be available."
"\n Did you forget ..."
"Please specify a valid backend using 'options(repr.plot.backends = c(
)
```
(or, instead of available, you could say 'working' or whatever you think is the most accurate).